spec/views/posts/show.html.erb_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
```

Comment:
Great job team 🤝

- [ ] [Optional] I would recommend that you add the missing tests as specified in the project requirements.
![posts-show](https://user-images.githubusercontent.com/63917725/216164529-72925223-9305-4658-bfb1-a654bd6ac3a5.PNG)